pkg/kubelet/kuberuntime/kuberuntime_sandbox.go

```diff
@@ -25,8 +26,28 @@ import (
 	kubecontainer "k8s.io/kubernetes/pkg/kubelet/container"
 	"k8s.io/kubernetes/pkg/kubelet/network"
 	"k8s.io/kubernetes/pkg/kubelet/types"
+	"k8s.io/kubernetes/pkg/kubelet/util/format"
 )
 
+// createPodSandbox creates a pod sandbox and returns (podSandBoxID, message, error).
+func (m *kubeGenericRuntimeManager) createPodSandbox(pod *api.Pod, podIP string, attempt uint32) (string, string, error) {
```

Comment:
I think this will continue to be required before the network plugin is fully moved to CRI.